packages/main/src/plugin/testutils/fake-informer.ts

```diff
@@ -0,0 +1,64 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ErrorCallback, KubernetesObject, ObjectCallback } from '@kubernetes/client-node';
+
+export class FakeInformer {
```

Comment:
hello, is that this class will be used in other places than kubernetes-context-state ?

Reply:
For this feature no, but it is very generic and could be useful for any other code using informers

Reply:
I would keep the objet in the kubernetes-context-state.spec.ts and only if we need it elsewhere we would move it rather than creating a `testutils` directory with an hypothetical reuse of this object.

Also